test/functional/services/remote/webdriver.ts

```diff
@@ -125,6 +125,7 @@ async function attemptToCreateCommand(
         });
         chromeCapabilities.set('unexpectedAlertBehaviour', 'accept');
         chromeCapabilities.set('goog:loggingPrefs', { browser: 'ALL' });
+        chromeCapabilities.set('acceptInsecureCerts', true);
```

Comment:
Do we need a similar change for the `msedge` and `firefox` cases, or are we only targeting `chrome` here?

Reply:
I think so https://developer.mozilla.org/en-US/docs/Web/WebDriver/Capabilities/acceptInsecureCerts

Reply:
I didn't add it since we don't run *all* tests in Firefox. But okay, I will add this suit in Firefox tests as well.